pkg/providers/docker/config/template-cp.yaml

```diff
@@ -201,7 +204,10 @@ spec:
 {{- if .kubeletExtraArgs }}
 {{ .kubeletExtraArgs.ToYaml | indent 10 }}
 {{- end }}
```

Comment:
This is it? Seems like an else would be nice here, but not a deal breaker for me